src/PanelTraits/Update.php

```diff
@@ -73,4 +74,22 @@ public function getUpdateFields($id)
 
         return $fields;
     }
+
+    private function getEntryValue($entry, $field)
+    {
+        if (isset($field['entity'])) {
+            $relationArray = explode('.', $field['entity']);
+            $relatedModel = array_reduce(array_splice($relationArray, 0, -1), function ($obj, $method) {
+                return $obj->{$method} ? $obj->{$method} : $obj;
+            }, $entry);
+
+            if ($relatedModel->{end($relationArray)} && $relatedModel->{end($relationArray)}() instanceof HasOneOrMany) {
```

Comment:
Without the first check, the second one could throw an error if the ```$model->relationMethod()``` doesn't exist in the model. And I agree, it's not that readable.
I think that by replacing the first call with ```method_exists($relatedModel, end($relationArray))``` we could improve the readability. Would you agree?